drivers/overlay/ovmanager/ovmanager.go

```diff
@@ -57,7 +57,7 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		config:   config,
 	}
 
-	d.vxlanIdm, err = idm.New(nil, "vxlan-id", vxlanIDStart, vxlanIDEnd)
```

Comment:
To avoid confusion, we should modify the constant `vxlanIDStart` value to 1 or remove the constant given it won't be used anywhere else in the `ovmanager` package.